KILTprotocol/sdk-js
# Mn 157 - doc public methods Attestation

## fixes #157  

## How to test:  
If you'd like to see how the doc looks like:  
Run `npm run build:docs`. This generates the doc at `sdk-js/docs/api`.
Open the so-generated files in your browser: `sdk-js/docs/api/classes/attestation.attestation-1.html`.

## Checklist:

- [x] I have verified that the code works // tests ran
